### DISCUSSION TITLE
Link to section in README doesn’t work

### DISCUSSION THREAD

**Discussion by alonfaraj** [community/community#60722] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.121.0 (104034577)
Device: iPhone 12 (iOS 16.5.1)
</details>

When a link in the README leads to a section in the same README, click on it only refresh the page and go to the top. 
It works fine on the web version.

**Comment by Rohan-San** [comment, marked as answer]
The section headings should not contain any special characters. And the link referencing said section should be hyphen separated and in lowercase.

[This is an example] (#section-heading)
# Section Heading
